chapters/102/index.md

```diff
@@ -0,0 +1,124 @@
+# Jina 102
```

Comment:
For the purpose of a readable table of contents, should we add something like:

_Jina 102 - Understanding how basic components work together?_

What do you think?

Reply:
I'm down with that. Maybe even simpler: *Jina 102: How basic components work together*

Reply:
Your the docs guy, you can choose which is best :)

Reply:
@alexcg1 in this case we can adjust both 101 & 102 so it's clear what people could learn just by looking at the titles?

Jina 101: Basic components 
Jina 102: How basic components work together

Reply:
I love that @pswu11! Let's go with that for both of them!